### ISSUE TITLE
Improve article list metadata

### ISSUE THREAD

**Issue by IamJoseph** [markerikson/react-redux-links#70]
In order to provide the most up-to-date information for react-redux links metadata should be included for the articles. My suggestion would be to add the dates the articles were published. If this is something you're interested in let me know and I can submit a PR.

**Comment by markerikson** [comment]
Seems like a worthwhile idea.  My main question is formatting.  Here's what my current entry format looks like:

Markdown:
```
- **Article Title**  
  http://link.to/article  
  Article description here
```

Result:

- **Article Title**  
  http://link.to/article  
  Article description here

I'm not sure exactly how I'd like to see the date squeezed into that.  Ideas?

**Comment by Sprinter100** [comment]
In parenthesis, after a titile with a dot.

- **Article Title. (10.02.17)**  
  http://link.to/article  
  Article description here

**Comment by markerikson** [comment]
I'm a firm believer in [ISO-8601 as the One True Way to write dates](https://xkcd.com/1179/) :)

**Comment by IamJoseph** [comment]
lol so should I proceed with @Sprinter100's suggestion with ISO-8601 formatting?

**Comment by markerikson** [comment]
Yeah, parentheses at the end of the article title should work, although maybe not bolded:

- **Article Title** (2017-10-02)
  http://example.com/article
  Article description here
